bokeh/server/configure.py

```diff
@@ -88,7 +88,7 @@ def configure_flask(config_argparse=None, config_file=None, config_dict=None):
 def register_blueprint():
     global registered
     if registered:
-        warnings.warn(
+        logging.warn(
```

Comment:
The warning will not show up if logging is not configured. I imagine `warnings.warn` was intended so that this message always makes it to users.